skfda/exploratory/visualization/_ddplot.py

```diff
@@ -72,41 +81,26 @@ def plot(
         distributions,one in each axis. It is useful to understand how
         our data is more related with one subset of data / distribution
         than another one.
-        Args:
-            chart: figure over with the graphs are plotted or axis over
-                where the graphs are plotted. If None and ax is also
-                None, the figure is initialized.
-            fig: figure over with the graphs are plotted in case ax is not
-                specified. If None and ax is also None, the figure is
-                initialized.
-            ax: axis where the graphs are plotted. If None, see param fig.
-            kwargs: if dim_domain is 1, keyword arguments to be passed to the
-                matplotlib.pyplot.plot function; if dim_domain is 2, keyword
-                arguments to be passed to the matplotlib.pyplot.plot_surface
-                function.
         Returns:
             fig (figure object): figure object in which the depths will be
             scattered.
         """
+        self.artists = np.array([])
         margin = 0.025
         width_aux_line = 0.35
         color_aux_line = "gray"
 
-        fig, axes = _get_figure_and_axes(chart, fig, ax)
-        fig, axes = _set_figure_layout_for_fdata(
-            self.fdata, fig, axes,
-        )
-
-        ax = axes[0]
+        ax = self.axes[0]
 
-        ax.scatter(
-            self.depth_dist1,
-            self.depth_dist2,
-            **kwargs,
-        )
+        for d1, d2 in zip(self.depth_dist1, self.depth_dist2):
+            self.artists = np.append(self.artists, ax.scatter(
+                d1,
+                d2,
+                picker=2,
+            ))
 
         # Set labels of graph
-        fig.suptitle("DDPlot")
+        ax.set_title("DDPlot")
```

Comment:
Probably better to use the dataset name. I am not opposed to adding an optional `name` parameter for `BasePlot` though.